static/openapi/prerendering-spec.json

```diff
@@ -3060,1025 +3384,1091 @@
+                      }
+                    ]
+                  }
                 }
               }
             }
           },
           "400": {
-            "description": "Bad request",
+            "description": "Bad request.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Bad request",
+                  "description": "Bad request.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Header x-photoniq-customerid is required"
+                  }
                 }
               }
             }
           },
           "500": {
-            "description": "Internal server error",
+            "description": "Internal server error.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Internal server error",
+                  "description": "Internal server error.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Internal server error"
+                  }
                 }
               }
             }
           }
         }
       }
     },
-    "/api/prerender/v1/origins/{origin}/prefetch/configs": {
-      "get": {
-        "summary": "Get prefetch configuration",
-        "tags": ["Prefetch Configurations"],
-        "description": "Returns the prefetch configuration associated with the origin.",
+    "/api/prerender/v1/origins/{origin}/prefetch/triggers": {
+      "post": {
+        "summary": "Create prefetch trigger",
+        "tags": ["Prefetch"],
+        "description": "Creates a prefetch trigger associated with the origin.",
```

Comment:
Do people know what a _prefech trigger_ means?

Reply:
This will probably be explained better in the general prerendering documentation. 
Any thoughts, @diana-macrometa?

Reply:
Yes, but we need to write that, because it currently does not exist.